cos-fleetshard-sync-it/src/test/java/org/bf2/cos/fleetshard/sync/it/NamespaceProvisionerBadIdTest.java

```diff
@@ -0,0 +1,138 @@
+package org.bf2.cos.fleetshard.sync.it;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+import javax.inject.Inject;
+import javax.ws.rs.core.MediaType;
+
+import org.bf2.cos.fleet.manager.model.ConnectorNamespaceTenantKind;
+import org.bf2.cos.fleetshard.api.ManagedConnectorCluster;
+import org.bf2.cos.fleetshard.support.resources.Namespaces;
+import org.bf2.cos.fleetshard.support.resources.Resources;
+import org.bf2.cos.fleetshard.sync.FleetShardSyncConfig;
+import org.bf2.cos.fleetshard.sync.client.FleetShardClient;
+import org.bf2.cos.fleetshard.sync.it.support.OidcTestResource;
+import org.bf2.cos.fleetshard.sync.it.support.SyncTestProfile;
+import org.bf2.cos.fleetshard.sync.it.support.SyncTestSupport;
+import org.bf2.cos.fleetshard.sync.it.support.WireMockServer;
+import org.bf2.cos.fleetshard.sync.it.support.WireMockTestResource;
+import org.eclipse.microprofile.config.ConfigProvider;
+import org.junit.jupiter.api.Test;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.github.tomakehurst.wiremock.client.WireMock;
+import com.github.tomakehurst.wiremock.http.ContentTypeHeader;
+import com.github.tomakehurst.wiremock.http.RequestMethod;
+
+import io.fabric8.kubernetes.api.model.Namespace;
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.TestProfile;
+
+import static io.restassured.RestAssured.given;
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.bf2.cos.fleetshard.support.resources.Resources.uid;
+
+@QuarkusTest
+@TestProfile(NamespaceProvisionerBadIdTest.Profile.class)
+public class NamespaceProvisionerBadIdTest extends SyncTestSupport {
```

Comment:
Guess this could have been just a change to https://github.com/bf2fc6cc711aee1a0c2a/cos-fleetshard/blob/2c8ec7165431998bd113630c7939a586ab6c8c41/cos-fleetshard-sync-it/src/test/java/org/bf2/cos/fleetshard/sync/it/NamespaceProvisionerTestBadName.java?

Reply:
probably, I'll do a refactor in a separate PR if ok